packages/client/modules/userDashboard/components/OrgBilling/OrgFeatureFlags.tsx

```diff
@@ -39,14 +39,6 @@ const FeatureNameGroup = styled('div')({
   }
 })
 
-// TODO: create a migration that updates featureName to be a readable string
-// then update the references throughout the app and remove this
-const FEATURE_NAME_LOOKUP: Record<string, string> = {
```

Comment:
This is no longer needed. When adding feature flags in the future, the flag name should be human readable so that we can show it in the client. 

Alternatively, we can have `featureName` and add `readableFeatureName` to `FeatureFlag` or re-add the lookup in the client.

Reply:
-1 We still have `insights` active, which is not very human friendly

Reply:
True! I've added that back, and will suggest we remove the insights flag in Slack